File: 백준/Bronze/11720. 숫자의 합/숫자의 합.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int num = sc.nextInt();
        String numStr = sc.next();
        int answer = 0;

        for (int i = 0; i < num; i++) answer += Integer.parseInt(String.valueOf(numStr.toCharArray()[i]));

        System.out.println(answer);

    }
}

File: 백준/Bronze/11720. 숫자의 합/숫자의 합.kt
import java.util.Scanner

fun main() {
    val sc = Scanner(System.`in`)
    val num = sc.nextInt()
    val numStr = sc.next()
    val answer = numStr.sum()

    print(answer)
}

private fun String.sum(): Int {
    var result = 0
    forEach { result += it.digitToInt() }
    return result
}